music_assistant/controllers/streams.py

```diff
@@ -1021,7 +1022,13 @@ async def get_queue_item_stream(
         elif streamdetails.volume_normalization_mode == VolumeNormalizationMode.MEASUREMENT_ONLY:
             # volume normalization with known loudness measurement
             # apply volume/gain correction
-            gain_correct = streamdetails.target_loudness - streamdetails.loudness
+            if streamdetails.prefer_album_loudness and streamdetails.loudness_album not in [
+                inf,
+                -inf,
+            ]:
```

Comment:
```suggestion
            if streamdetails.prefer_album_loudness and streamdetails.loudness_album not in (
                inf,
                -inf,
            ):
```
let's use a tuple here. Also I wonder I you got an invalid measurement in the db.

Reply:
> Also I wonder I you got an invalid measurement in the db.

How so. What looks suspicious to you?